tests/odh/validating_admission_policy_test.go

```diff
@@ -70,6 +71,10 @@ func TestValidatingAdmissionPolicy(t *testing.T) {
 	ns = CreateTestNamespaceWithName(test, uniqueSuffix("vap"))
 	defer test.Client().Core().CoreV1().Namespaces().Delete(test.Ctx(), ns.Name, metav1.DeleteOptions{})
 
+	// Create a namespace that will not receive the `kueue.x-k8s.io/queue-name` label
+	nsNoLabel = CreateTestNamespaceWithName(test, uniqueSuffix("vap"))
```

Comment:
This way we can distinguish between the two namespaces in the cluster:
```suggestion
	nsNoLabel = CreateTestNamespaceWithName(test, uniqueSuffix("vap-no-label"))
```